src/components/Page/Page.jsx

```diff
@@ -25,6 +25,7 @@ class Page extends React.Component {
   }
 
   componentDidMount() {
+    window.scrollTo(0,0);
```

Comment:
What about anchor links?

e.g. `page#section` ?

Reply:
What about if you reopen the last closed tab?

Reply:
Yeah this approach isn't good enough. What if we bind onto menu link?

Reply:
Anchor links work fine, for last closed tab, I'm thinking of storing scroll positions when navigating the code and restoring it back to the stored position. This way if a user has read upto a certain position, his/her position will be preserved when he/she comes back to the same page.